localstack/services/transcribe/provider.py

```diff
@@ -136,7 +140,16 @@ def start_transcription_job(
         if not output_key:
             output_key = f"{job_name}.json"
 
-        transcript = Transcript(TranscriptFileUri=f"s3://{output_bucket}/{output_key}")
+        s3_client = connect_to().s3
+
+        # the presign url is valid for 15 minutes
+        presign_url = s3_client.generate_presigned_url(
+            "get_object",
+            Params={"Bucket": output_bucket, "Key": output_key},
+            ExpiresIn=60 * 15,
+        )
```

Comment:
I think we should re-generate the `TranscriptFileUri` for `GetTranscriptionJob` too so that we extend the validity of the pre-signed URL

Reply:
Thanks @bentsku. Have made the required changes.

Reply:
Awesome, thanks!